s3/src/main/scala/com/meltwater/fawn/s3/S3Types.scala

```diff
@@ -0,0 +1,100 @@
+package com.meltwater.fawn.s3
+
+import cats.implicits._
+import com.lucidchart.open.xtract.{__, XmlReader}
+import enumeratum._
+
+case class Owner(displayName: String, id: String)
+
+object Owner {
+  implicit val xmlDecoder: XmlReader[Owner] = (
+    (__ \ "DisplayName").read[String],
+    (__ \ "ID").read[String]
+  ).mapN(apply)
+}
+
+case class Initiator(displayName: String, id: String)
+
+object Initiator {
+  implicit val xmlDecoder: XmlReader[Initiator] = (
+    (__ \ "DisplayName").read[String],
+    (__ \ "ID").read[String]
+  ).mapN(apply)
+}
+
+case class Bucket(creationDate: String, name: String)
+
+object Bucket {
+  implicit val xmlDecoder: XmlReader[Bucket] = (
+    (__ \ "CreationDate").read[String],
+    (__ \ "Name").read[String]
+  ).mapN(apply)
+}
+
+case class Object(
```

Comment:
this name may be too general, scala does have the `object` keyword so this is a bit close for me, consider `S3Object`